apps/web/playwright/lib/appstore/wipe-my-cal.test.ts

```diff
@@ -0,0 +1,66 @@
+import { expect } from "@playwright/test";
+import dayjs from "dayjs";
+
+import { test } from "../../lib/fixtures";
+
+test.describe.configure({ mode: "parallel" });
+
+test.describe("Wipe my Cal App Test", () => {
+  test("Browse upcoming bookings and validate button shows and triggering wipe my cal button", async ({
+    page,
+    users,
+    bookings,
+  }) => {
+    const pro = await users.create();
+    const [eventType] = pro.eventTypes;
+    await prisma?.credential.create({
+      data: {
+        key: {},
+        type: "wipemycal_other",
+        userId: pro.id,
```

Comment:
appId is still missing here

```suggestion
        userId: pro.id,
        appId: "wipemycal",
```